src/storage/hummock_sdk/src/lib.rs

```diff
@@ -266,3 +267,60 @@ pub fn version_checkpoint_path(root_dir: &str) -> String {
 pub fn version_checkpoint_dir(checkpoint_path: &str) -> String {
     checkpoint_path.trim_end_matches(|c| c != '/').to_string()
 }
+
+#[derive(Clone, Copy, PartialEq, Eq, Hash, Default, Debug, PartialOrd, Ord)]
+pub struct EpochWithGap(u64);
```

Comment:
nits: it is better to add some doc here to explain why we need this struct, when this struct will be used, and what the different parts are in the u64